actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+*   Fix adding implicitly rendered template digests to ETags
+
```

Comment:
Thanks for catching that @prathamesh-sonpatki. I amended this commit and added a full stop.